<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>772</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>10</number>
     </property>
     <property name="usesScrollButtons">
      <bool>true</bool>
     </property>
     <property name="movable">
      <bool>true</bool>
     </property>
     <property name="tabBarAutoHide">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Basico</string>
      </attribute>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>301</width>
         <height>91</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Introduce carácter</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLineEdit" name="lineEdit_Simb"/>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_Simb2"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_CrearAFN">
          <property name="text">
           <string>Crear AFN</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Union</string>
      </attribute>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>120</y>
         <width>330</width>
         <height>70</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Union de automatas</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QComboBox" name="comboBox_AFNs"/>
            </item>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Union:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox_AFNs2"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_Union">
            <property name="text">
             <string>Unir</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Conca</string>
      </attribute>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>60</y>
         <width>405</width>
         <height>64</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Concatenación de automatas</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QComboBox" name="comboBox_AFNs3"/>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Concatenar:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox__AFNs4"/>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_Conca">
            <property name="text">
             <string>Concatenar</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Cerradura+</string>
      </attribute>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>60</y>
         <width>260</width>
         <height>64</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Cerradura +</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QComboBox" name="comboBox_AFNs5"/>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_CerrMAS">
            <property name="text">
             <string>Aplicar Cerradura+</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>Kleene</string>
      </attribute>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>60</y>
         <width>233</width>
         <height>64</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Cerradura de Kleene</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QComboBox" name="comboBox_AFNs6"/>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_Kleene">
            <property name="text">
             <string>Aplicar Kleene</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_6">
      <attribute name="title">
       <string>Opcional?</string>
      </attribute>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>80</y>
         <width>252</width>
         <height>64</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Opcional ?</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QComboBox" name="comboBox_AFNs7"/>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_Opcional">
            <property name="text">
             <string>Aplicar Opcional?</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_7">
      <attribute name="title">
       <string>Union Especial</string>
      </attribute>
      <widget class="QTableWidget" name="Table1">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>651</width>
         <height>271</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_UnirEsp">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>320</y>
         <width>151</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>Aplicar Union Especial</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_8">
      <attribute name="title">
       <string>AFN -&gt; AFD</string>
      </attribute>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>711</width>
         <height>371</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout" columnstretch="7,20">
        <item row="0" column="1">
         <widget class="QTableWidget" name="Table2"/>
        </item>
        <item row="0" column="0">
         <widget class="QToolBox" name="toolBox">
          <property name="currentIndex">
           <number>1</number>
          </property>
          <widget class="QWidget" name="page">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>100</width>
             <height>30</height>
            </rect>
           </property>
           <attribute name="label">
            <string>Convertir AFN a AFD</string>
           </attribute>
           <widget class="QPushButton" name="pushButton_ConvertirAFN">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>100</y>
              <width>71</width>
              <height>32</height>
             </rect>
            </property>
            <property name="text">
             <string>Convertir</string>
            </property>
           </widget>
           <widget class="QWidget" name="layoutWidget">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>30</y>
              <width>161</width>
              <height>61</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <item>
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>Selecciona AFN</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox_AFNs8"/>
             </item>
            </layout>
           </widget>
          </widget>
          <widget class="QWidget" name="page_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>181</width>
             <height>267</height>
            </rect>
           </property>
           <attribute name="label">
            <string>Archivo AFD</string>
           </attribute>
           <widget class="QWidget" name="layoutWidget">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>20</y>
              <width>153</width>
              <height>24</height>
             </rect>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_11">
             <item>
              <widget class="QLabel" name="label_12">
               <property name="text">
                <string>Guardar AFD actual</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="toolButton_Guardar">
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="layoutWidget">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>80</y>
              <width>151</width>
              <height>24</height>
             </rect>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_12">
             <item>
              <widget class="QLabel" name="label_13">
               <property name="text">
                <string>Cargar AFD</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="toolButton_CargaAFD">
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
          <widget class="QWidget" name="page_4">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>100</width>
             <height>30</height>
            </rect>
           </property>
           <attribute name="label">
            <string>AFD</string>
           </attribute>
           <widget class="QWidget" name="layoutWidget">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>30</y>
              <width>167</width>
              <height>64</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_8">
             <item>
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>Selecciona AFD para ver</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_10">
               <item>
                <widget class="QComboBox" name="comboBox_AFDs1"/>
               </item>
               <item>
                <widget class="QPushButton" name="pushButton_verAFD">
                 <property name="text">
                  <string>Ver</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="layoutWidget">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>120</y>
              <width>161</width>
              <height>42</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_9">
             <item>
              <widget class="QLabel" name="label_11">
               <property name="text">
                <string>AFD actual: </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_AFDactual">
               <property name="text">
                <string>none</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_9">
      <attribute name="title">
       <string>Analizador Lexico</string>
      </attribute>
      <widget class="QTableWidget" name="Table3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>321</width>
         <height>381</height>
        </rect>
       </property>
      </widget>
      <widget class="QWidget" name="">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>120</y>
         <width>171</width>
         <height>111</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_10">
        <item>
         <widget class="QLineEdit" name="lineEdit_Lexic"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_UsarLexic">
          <property name="text">
           <string>Usar Analizador</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_10">
      <attribute name="title">
       <string>ER -&gt; AFN</string>
      </attribute>
      <widget class="QPushButton" name="pushButton_CreaER">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>80</y>
         <width>121</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>Crear ER -&gt; AFN</string>
       </property>
      </widget>
      <widget class="QWidget" name="">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>40</y>
         <width>261</width>
         <height>71</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_11">
        <item>
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>Expresión Regular</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_ERbox"/>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_11">
      <attribute name="title">
       <string>Calcu</string>
      </attribute>
      <widget class="QWidget" name="">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>70</y>
         <width>421</width>
         <height>91</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_13">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_12">
          <item>
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>Introduzca la expresión</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_Calcu"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_IniciarEval">
          <property name="text">
           <string>Iniciar</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
